config/sample-bigtreetech-hermit-crab-canbus.cfg

```diff
@@ -0,0 +1,128 @@
+# This file contains common pin mappings for the BIGTREETECH HermitCrab
+# Canbus board. To use this config, the firmware should be compiled for the
+# STM32F072 with "8 MHz crystal" and USB (on PA11/PA12) or CAN bus (on PB8/PB9).
+# The "HermitCrab" micro-controller will be used to control the components on the nozzle.
+
+[mcu HermitCrab]
+#canbus_uuid: 0e0d81e4210c
+#serial: /dev/serial/by-id/usb-Klipper_Klipper_firmware_12345-if00
+
+[adxl345]
+cs_pin: HermitCrab: PB12
+spi_software_sclk_pin: HermitCrab: PB13
+spi_software_mosi_pin: HermitCrab: PB15
+spi_software_miso_pin: HermitCrab: PB14
+axes_map: y,z,-x
+#   The accelerometer axis for each of the printer's x, y, and z axes.
+#   This may be useful if the accelerometer is mounted in an
+#   orientation that does not match the printer orientation. For
+#   example, one could set this to "y,x,z" to swap the x and y axes.
+#   It is also possible to negate an axis if the accelerometer
+#   direction is reversed (eg, "x,z,-y"). The default is "x,y,z".
+#rate: 3200
+#   rates: 3200, 1600, 800, 400, 200, 100, 50, and 25
+
+[resonance_tester]
+accel_chip: adxl345
+probe_points:
+    100,100,20  # an example
+
+[extruder]
+step_pin: HermitCrab: PA6
+dir_pin: !HermitCrab: PA7
+enable_pin: !HermitCrab: PA5
+microsteps: 16
+rotation_distance: 3.45
```

Comment:
The rotation_distance seems to be too small: it is typically an order of magnitude bigger. Not that it matters too much for the sample, and the users are expected to update it anyway, but maybe it would be great to specify a more realistic value from the start?

Reply:
Yeah, 3.45 is for H2 extruder.  It has been changed to 33.5 for Common extruders

Reply:
Ah, I see, my bad. Then please feel free to revert the change and keep 3.45 as the rotation distance.